vars/sectionDeployToAKS.groovy

```diff
@@ -257,8 +259,44 @@ def call(params) {
               }
             }
           }
+
+        } else {
+
+          onFunctionalTestEnvironment(environment) {
+            // NB: non-service apps have no URL of their own: therefore require an alternative test URL when
+            //     running functional tests.
+            if (config.fullFunctionalTest && config.fullFunctionalTestNonServiceAppTestUrl != null) {
```

Comment:
Would recommend try and reuse https://github.com/hmcts/cnp-jenkins-library/pull/987/files#diff-ea3aab027b617efc21c3b315b057a003b215bad8d524eb23684fe77514b4b53fR147 than defining everything here.